app/presenters/document_presenter.rb

```diff
@@ -12,7 +12,7 @@ def to_json
       base_path: document.base_path,
       title: document.title,
       description: document.summary,
-      format: document.format,
```

Comment:
I've had a chat with some of the Publishing Platform team about this. I have 2 options:

- split the Specialist Document schema into numerable schemas
- filer on the `document_type` attribute that is sent as part of the metadata

As a result of the chat, I'm going to go with the latter, but won't be able to do it on the Publishing API side fully until we implement Elastic Search. Until that happens, I'm going to go with a temp fix and throw away all the Specialist Document formats I don't need.